source/agora/node/Ledger.d

```diff
@@ -500,6 +502,48 @@ public class Ledger
         }
     }
 
+    /***************************************************************************
+
+        Checks whether the `tx` is an acceptable double spend transaction.
+
+        If `tx` is not a double spend transaction, then returns true.
+        If `tx` is a double spend transaction, and its fee is considerable higher
+        than the existing double spend transactions, then returns true.
+        Otherwise this function returns false.
+
+        Params:
+            tx = transaction
+            threshold_pct = percentage by which the fee of the new transaction has
+              to be higher, than the previously highest double spend transaction
+
+        Returns:
+            whether the `tx` is an acceptable double spend transaction
+
+    ***************************************************************************/
+
+    public bool isAcceptableDoubleSpent(in Transaction tx, ubyte threshold_pct) @safe
+    {
+
+        Amount tx_fee;
+        if (fee_man.getAdjustedTXFee(tx, &utxo_set.peekUTXO, tx_fee) != null)
+            return false;
+
+        // only consider a double spend transaction, if its fee is
+        // considerably higher than the current highest fee
+        auto fee_threshold = getDoubleSpentHighestFee(tx);
+
+        // if the fee_threshold is null, it means there won't be any double
+        // spend transactions, after this transaction is added to the pool
+        if (!fee_threshold.isNull())
+            fee_threshold.percentage(threshold_pct + 100);
```

Comment:
This line, along with 541, triggers a new deprecation message. It should call `.get()`